packages/cxl-ui/package.json

```diff
@@ -15,12 +15,12 @@
     "@pika/plugin-build-web": "^0.9.2",
     "@pika/plugin-bundle-web": "^0.9.2",
     "@pika/plugin-standard-pkg": "^0.9.2",
-    "@vaadin/vaadin-accordion": "^2.0.0-alpha5",
-    "@vaadin/vaadin-button": "^3.0.0-alpha1",
-    "@vaadin/vaadin-context-menu": "^5.0.0-alpha2",
-    "@vaadin/vaadin-notification": "^2.0.0-alpha1",
-    "@vaadin/vaadin-progress-bar": "^2.0.0-alpha2",
-    "@vaadin/vaadin-tabs": "^4.0.0-alpha4",
+    "@vaadin/vaadin-accordion": "21.0.0-alpha3",
+    "@vaadin/vaadin-button": "21.0.0-alpha3",
+    "@vaadin/vaadin-context-menu": "21.0.0-alpha3",
+    "@vaadin/vaadin-notification": "21.0.0-alpha3",
+    "@vaadin/vaadin-progress-bar": "21.0.0-alpha3",
+    "@vaadin/vaadin-tabs": "21.0.0-alpha3",
     "@vaadin/vaadin-themable-mixin": "^1.6.2",
```

Comment:
But also, why hasn't `@vaadin/vaadin-themable-mixin` been bumped?

https://github.com/vaadin/web-components/tree/master/packages/vaadin-themable-mixin should be brought along for the ride, no?

Reply:
Agreed